instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/trace/ActivityLifecycleTrace.kt

```diff
@@ -40,10 +39,9 @@ internal class ActivityLifecycleTrace : AppCompatActivity() {
         val trackingConsent = intent.getTrackingConsent()
 
         Datadog.initialize(this, credentials, config, trackingConsent)
-        Datadog.registerFeature(
-            LogsFeature.Builder().useCustomEndpoint(RuntimeConfig.logsEndpointUrl).build()
```

Comment:
Did we lose setting the custom endpoint for the logs feature? I don't see where it moved to.

Reply:
No, we didn't, because `RuntimeConfig.logsFeatureBuilder()` calls `useCustomEndpoint` under the hood.